api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java

```diff
@@ -185,14 +185,14 @@ private static UserContainerInitializer findUserContainerInitializer() {
      * </p>
      * <p>
      * <p>
-     * Scanning may also have negative impact on bootstrap performance.
+     * Scanning may also have negative impact on se performance.
```

Comment:
'se performance' does not make sense. I would prefer not to change bootstrap to se in this sentence.